bindings/jupyroot/CMakeLists.txt

```diff
@@ -42,9 +42,10 @@ foreach(val RANGE ${how_many_pythons})
   # Set the suffix to '.so' and the prefix to 'lib'
   set_target_properties(${libname} PROPERTIES  ${ROOT_LIBRARY_PROPERTIES})
   if(MSVC)
+    list(GET python_libraries ${val} python_library)
     set_target_properties(${libname} PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
     set_target_properties(${libname} PROPERTIES SUFFIX ".pyd")
-    target_link_libraries(${libname} PUBLIC Core)
+    target_link_libraries(${libname} PUBLIC Core  ${python_library})
```

Comment:
So we are linking `libJupyROOT` with `libpython` again. Do you understand why this is necessary for `libJupyROOT` on Windows but not for `libROOTPythonizations` or `libcppyy`? In the end, the three libraries are extension modules that are used from Python: `import libname`.

Reply:
No idea, for now I just want to make it build successfully. I'll check later what is really needed and how to make it fully working on Windows...